test/hotspot/jtreg/vmTestbase/nsk/share/native/native_thread.cpp

```diff
@@ -98,7 +98,11 @@ void* procedure(void* t) {
     thread->started  = 1;
     thread->status   = thread->procedure(thread->context);
     thread->finished = 1;
+#ifdef windows
```

Comment:
Lower case 'windows' ??  Not an issue for your change, but odd.

Reply:
Yeah, it's a bit odd.  It's defined up at the top of the file, based on whether `_WIN32` is defined.  I've no
idea why `_WIN32` isn't just used instead.